spec/scoreSpec.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+console.log("I AM IN THE SPEC!")
+
+describe('Score', () => {
+
+  var score 
+  var game 
+
+  beforeEach(function(){
+    score = new Score
+    game = new Game // How to spy this? 
```

Comment:
i usually spy functions instead of classes. I think at the first it's good start creating objects and then spy on them